CoreAPI/src/test/java/com/bae/TestConstants.java

```diff
@@ -0,0 +1,133 @@
+package com.bae;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.bae.domain.Associate;
+import com.bae.domain.AtmTransactionLocations;
+import com.bae.domain.Citizen;
+import com.bae.domain.EposTransactionLocations;
+import com.bae.domain.MobileCallRecords;
+import com.bae.domain.PeopleBankCard;
+import com.bae.domain.PeopleMobile;
+import com.bae.domain.VehicleLocations;
+import com.bae.domain.VehicleRegistration;
+
+
+public class TestConstants {
+
+	//Citizen Controller Test
+	public static final Citizen MOCK_CITIZEN = new Citizen(12348L, "Crazy", "Ivan",
+			"543 Crazy Street, Crazy Town, CR 4ZY", LocalDate.of(1990, 01, 01), "Russiatown", "Male");
+
+
+	public static final String CITIZEN_FORENAME = "http://citizen:8003/getCitizensByForenames/Crazy";
+	public static final String CITIZEN_SURNAME = "http://citizen:8003/getCitizensBySurname/Ivan";
```

Comment:
Might be worth extracting repeated sections into their own variables.
i.e. public static final String CITIZEN_URL = "http://citizen:8003"